### ISSUE TITLE
Description is the same like in QID-2548-MVAQnetClusKmeans, so it's unclear what is the difference between these 2 q.

### ISSUE THREAD

**Issue by polarstern** [QuantLet/QID-2545-MVAQnetClusKmeansT#3]
Description is the same like in QID-2548-MVAQnetClusKmeans, so it's unclear what the difference between these 2 q is.

**Comment by anastasia-stepanchenko** [comment]
I've done it a little bit more clear in both descriptions.

**Comment by polarstern** [comment]
Ok, please add some info about results - that in one q there is 3 MDS and in another 2 MDS - and describe which MDS are given.

**Comment by anastasia-stepanchenko** [comment]
this is already in "example" part of metainfo, do you want it to be in description?

**Comment by polarstern** [comment]
yes, because now the only difference is one word - "squared". Leave examples as well.

**Comment by anastasia-stepanchenko** [comment]
gemacht

**Comment by polarstern** [comment]
gut gemacht! :-)

**Comment by lborke** [comment]
Hi all,
actually the only real difference between these two Qs is that QID-2548 is plotting for MVA and BCS and QID-2545 (this one) is plotting additionally for all Qlets. The issue about "squared distance" was (of course) a bug. Now I have improved it.This version (QID-2545) was initially uploaded by me. So in case of doubt adapt my style/version like in this Q.

I have improved some details in the style of this Q. But the output/pictures remain the same.
